book/_config.yml

```diff
@@ -1,14 +1,14 @@
-title: Template
-author: TeachBooks Team, built with <a href="https://teachbooks.tudelft.nl/">TeachBooks</a> and <a href="https://jupyterbook.org/">Jupyter Book</a>, <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="license noopener noreferrer"">CC BY 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg" alt=""></a> #add your own name here
-logo: figures/TB_logo.png #add your own logo here
+title: eurocsdms eWaterCycle workshop 
+author: Rolf Hut (Delft University of Technology) & the ever growing eWaterCycle team (Netherlands eScience Center, Delft University of Technology), built with <a href="https://teachbooks.tudelft.nl/">TeachBooks</a> and <a href="https://jupyterbook.org/">Jupyter Book</a>, <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="license noopener noreferrer"">CC BY 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg" alt=""></a> #add your own name here
+logo: figures/ewatercycle_logo.png #add your own logo here
 
 execute:
   execute_notebooks: "off"
 
-only_build_toc_files: true
+only_build_toc_files: false
```

Comment:
@MarkMelotto @RolfHut zag dit langskomen op github. Misschien beter om dit op `true` te laten staan vanwege alles bestanden in de "Deliverables" folder; deze worden nu allemaal als html paginas gerendered.

Daarnaast kan ook het grote aantal CSV en ipynb bestanden kan problemen opleveren met git: de git history zal er erg groot van worden en de repository clonen wordt langzamer. Het zou beter zijn om al deze niet essentiele bestanden ergens anders op te slaan (zenodo?)